### DISCUSSION TITLE
`CartesianGrid` is not working correctly for some `domain`

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [recharts/recharts#3556] [Q&A] [answered]
Hey there,

I'm having a problem with my chart when I set the `domain` from 0 to 30, the `24` line of the `CartesianGrid` is not showing:

<img width="524" alt="domain-30" src="https://bytebucket.org/201flaviosilva-labs/q-react-recharts-domain-problem/raw/c7e37c5f6fe57e660930e4a7da6b423a851e898a/imgs/domain-0-30.png">

But when I put `domain` from 0 to 40 it works normally:

<img width="521" alt="domain-40" src="https://bytebucket.org/201flaviosilva-labs/q-react-recharts-domain-problem/raw/c7e37c5f6fe57e660930e4a7da6b423a851e898a/imgs/domain-0-40.png">

Repo example: https://bitbucket.org/201flaviosilva-labs/q-react-recharts-domain-problem/

Am I doing something wrong?

Thanks any help :)

**Comment by ckifer** [comment, marked as answer]
I don't think you're doing anything incorrectly, I just think recharts' calculation for this isn't perfect. Thanks for raising visibility on it!

**Reply by 201flaviosilva** [reply]
ohhh ok ok thanks :)
